examples/gameroom/daemon/src/authorization_handler/error.rs

```diff
@@ -152,17 +154,24 @@ macro_rules! impl_from_sabre_errors {
     };
 }
 
-impl_from_sabre_errors!(
-    CreateContractActionBuildError,
-    CreateContractRegistryActionBuildError,
-    CreateNamespaceRegistryActionBuildError,
-    CreateNamespaceRegistryPermissionActionBuildError,
-    SabreProtoConversionError,
-    SabrePayloadBuildError
-);
+impl_from_sabre_errors!(AddressingError, ActionBuildError, SabrePayloadBuildError);
 
 impl From<SigningError> for AppAuthHandlerError {
     fn from(err: SigningError) -> Self {
         AppAuthHandlerError::SigningError(err.to_string())
     }
 }
+
+macro_rules! impl_from_transact_errors {
+    ($($x:ty),*) => {
+        $(
+            impl From<$x> for AppAuthHandlerError {
+                fn from(e: $x) -> Self {
+                    AppAuthHandlerError::TransactError(e.to_string())
+                }
+            }
+        )*
+    };
+}
+
+impl_from_transact_errors!(BatchBuildError, TransactionBuildError, ProtoConversionError);
```

Comment:
This macro doesn’t seem all that useful for one error

Reply:
Do you want it changed?

Reply:
That would be my preference -- the threshold for using any macros should be pretty high.

Reply:
Removed the macro and implemented the `From`s for each.